cmd/gateway/main.go

```diff
@@ -110,9 +110,8 @@ func cmdSetup(cmd *cobra.Command, args []string) (err error) {
 	}
 
 	overrides := map[string]interface{}{
-		"client.api-key":         setupCfg.APIKey,
-		"client.pointer-db-addr": setupCfg.SatelliteAddr,
-		"client.overlay-addr":    setupCfg.SatelliteAddr,
+		"client.api-key":        setupCfg.APIKey,
+		"client.satellite-addr": setupCfg.SatelliteAddr,
```

Comment:
Maybe, this can be removed, since it's already specified as `setupcfg`?

Reply:
How can I access `setupcfg` in `uplink/config.go`?

Reply:
Don't all the `setupCfg` flags get passed into `SaveConfigWithAllDefaults` automatically?

Reply:
Or maybe, if we remove the "setup" tags from apikey and satellite-addr, in GatewayFlags, then they get automatically propagated?

Reply:
I deleted `ApiKey` and `SatelliteAddr` from `UplinkFlags` and `GatewayFlags`. So these two configs are now available only in the `ClientConfig`. I also added a `noprefix` tag to them. When `noprefix = true` then the configs are bound to flags without the prefix, i.e. as `--api-key` and `--satellite-addr` instead of `--client.api-key` and `--client.satellite-addr`. See my latest commit.